add-image-and-video-dimensions.js

```diff
@@ -0,0 +1,45 @@
+const fs = require('fs');
+
+const glob = require('glob');
+const imageSize = require('image-size');
+const videoSize = require('get-video-dimensions');
+
+(async () => {
+  const files = glob.sync('src/**/*.{html,md}');
+  for (const file of files) {
+    const contents = fs.readFileSync(file, 'utf8').toString();
```

Comment:
Since you're in `async` context anyway, might be better to use `fs.promises.*` here and everywhere else.

You can just import them at the top as

```js
const fs = require('fs').promises;
```

and then use async APIs everywhere.

Additionally, you could extract file processing into a function to be able to handle files in parallel.

Reply:
I use the async file methods now, but have refrained from optimizing further. It's really just fast enough :-)